test/unit/spec/appWidgets/inputUtilsSpec.js

```diff
@@ -38,20 +38,26 @@ define(['widgets/appWidgets2/inputUtils', 'testUtil'], (InputUtils, TestUtil) =>
                 await TestUtil.waitForElement(document.body, 'div.modal-body', () =>
                     node.querySelector('button').click()
                 );
+                const expectedMsg = `Message id: ${msg.id}`;
                 expect(document.body.querySelector('div.modal-body').innerHTML).toContain(
-                    'Message id: ' + msg.id
+                    expectedMsg
                 );
                 await TestUtil.waitForElementState(
                     document.body,
                     () => {
-                        return document.body.querySelectorAll('.modal').length === 0;
+                        const elem = document.body.querySelector('div.modal-body');
```

Comment:
Make very explicit what we're looking for in case other tests leave a `.modal` element behind.

This really shouldn't be an issue with other changes, but I left it in as it's more explicit.